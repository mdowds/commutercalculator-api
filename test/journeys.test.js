const fetch = require('node-fetch');

let journeys;

describe('GET /journeys/to', () => {
    beforeAll(async () => {
        journeys = await fetch('http://localhost:5000/journeys/to/LBG').then(r => r.json());
    });

    it('should match snapshot', () => {
        expect(journeys).toMatchSnapshot();
    });
});