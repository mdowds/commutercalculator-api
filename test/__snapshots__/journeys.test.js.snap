// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /journeys/to should match snapshot 1`] = `
Object {
  "destination": Object {
    "id": "ACC",
    "name": "Acton Central",
    "position": Object {
      "lat": 51.5087577801896,
      "lng": -0.263430199363473,
    },
    "postcode": "W3 6BD",
    "zones": Array [
      2,
    ],
  },
  "results": Array [
    Object {
      "directionsUrl": "https://www.google.com/maps/dir/?api=1&origin=Abbey+Wood+Station,London&destination=Acton+Central+Station,London&travelmode=transit",
      "journeyTime": 110,
      "origin": Object {
        "id": "ABW",
        "name": "Abbey Wood",
        "position": Object {
          "lat": 51.4907840957031,
          "lng": 0.120271969782359,
        },
        "postcode": "SE2 9RH",
      },
    },
  ],
}
`;
